### DISCUSSION TITLE
How to handle built-in exception in Telebot

### DISCUSSION THREAD

**Discussion by g-ruby** [eternnoir/pyTelegramBotAPI#1726] [Q&A]
I have a similar error, how do I handle the error and if it is found, restart the bot. Or instead of this error, write something of your own, and display only it.

`2022-09-18 20:19:51,043 (async_telebot.py:324 MainThread) ERROR - TeleBot: "A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running"`

**Comment by coder2020official** [comment]
Use try/except when running polling.

**Reply by g-ruby** [reply]
Now work!
